### DISCUSSION TITLE
Legacy JDBC instantiation

### DISCUSSION THREAD

**Discussion by gordonthorpe** [dbeaver/dbeaver#8704] [Q&A] [answered]
Community Edition 7.0.4 has a check-box "Use legacy JDBC instantiation" on the Create New Driver dialog.  I have a driver that can't be loaded when this is selected.  What exactly does this do to load a driver?

**Comment by uslss** [comment, marked as answer]
Hi @gordonthorpe ,
in the recent version this checkbox is unchecked by default. It is needed for custom drivers only.
